mappings/src/androidMain/kotlin/com/revenuecat/purchases/kmp/mappings/SubscriptionOptions.android.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases.kmp.mappings
+
+import com.revenuecat.purchases.kmp.models.SubscriptionOptions
+import com.revenuecat.purchases.models.SubscriptionOptions as AndroidSubscriptionOptions
+
+internal fun AndroidSubscriptionOptions.toSubscriptionOptions(): SubscriptionOptions =
+    SubscriptionOptions(
+        basePlan = basePlan?.let { AndroidSubscriptionOption(it) },
```

Comment:
Note that this reverts the work that was done before to avoid calculating each of these at construction time by passing lambdas to the constructor. That didn't work when moving the `SubscriptionOptions` to the `models` module.

So I'm passing the calculated values here. Which is how it works in other hybrids. Lmk if you have concerns with this approach and we can think of a different solution...

Reply:
Nice one! Yeah, that makes sense, thanks!